docker-compose-integration.yaml

```
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   test:
-    image: elixir:latest
+    image: elixir:1.14
```

As I mentioned, the hex.pm images are better for running on CI.
```suggestion
    image: hexpm/elixir:1.14.4-erlang-25.3.2-alpine-3.16.5
```